substrate/frame/revive/src/wasm/mod.rs

```diff
@@ -87,19 +84,14 @@ pub struct CodeInfo<T: Config> {
 	refcount: u64,
 	/// Length of the code in bytes.
 	code_len: u32,
-	/// The API version that this contract operates under.
-	///
-	/// This determines which host functions are available to the contract. This
-	/// prevents that new host functions become available to already deployed contracts.
-	api_version: u16,
 	/// The behaviour version that this contract operates under.
 	///
 	/// Whenever any observeable change (with the exception of weights) are made we need
 	/// to make sure that already deployed contracts will not be affected. We do this by
 	/// exposing the old behaviour depending on the set behaviour version of the contract.
 	///
 	/// As of right now this is a reserved field that is always set to 0.
-	behaviour_version: u16,
+	behaviour_version: u32,
```

Comment:
Doesn't need migration. Everything was set to zero and we just widen `behavior_version` to cover both fields.

Reply:
65k  was not enough ? 🙃

Reply:
We would need a migration if keeping it `u16`. By widening it the encoded format of the structure stays the same.